.github/workflows/pr.yaml

```diff
@@ -94,7 +94,9 @@ jobs:
         uses: pnpm/action-setup@v2.1.0
         with:
           version: ${{ env.PNPM_VERSION }}
-          run_install: true
+          run_install: |
+            - recursive: true
+              args: [--no-frozen-lockfile]
```

Comment:
On the application is a must-have to install using lock files... why do you have to remove it?

Reply:
we had some problems with dependencies and node version. now with the refactor we might not need it anymore. I'll revert it and check if it works